applications/backend/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "extends": [
+      // "eslint:recommended",
```

Comment:
Do we need this in the future for the backend application? If not, please delete